### ISSUE TITLE
Give All Wearables Function?

### ISSUE THREAD

**Issue by puttichan** [kwsch/NHSE#473]
**Is your feature request related to a problem? Please describe.**
**A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]**

I am frustrated when I want to give myself only wearables, but NHSE right now can only give ALL or none, excluding fish/dive/etc.

**Describe the solution you'd like**
**A clear and concise description of what you want to happen.**

I wish there was a button to give all wearable ones, excluding fish/dive/etc.

Note: Requesting new editing features that *require research* are not Feature Requests. This is only for improving interactions and/or leveraging the current code in new/interesting ways.

**Describe alternatives you've considered**
**A clear and concise description of any alternative solutions or features you've considered.**

Maybe a way to compile it myself if this is a hassle?

**Comment by Manlibear** [comment]
I did something similar myself, for all furniture and wearables.

Didn't run into any issues compiling with Visual Studio though, everything worked out of the box, what happens when you compile it?

**Comment by nekkoclouds** [comment]
Yeah I agree, a function like this for all wearables and furniture separately would be great (with the furniture obviously not including things like the fossils, fish, bugs, art etc).  Or maybe only include fake art so that you have a reference in your catalog rather than having to google each piece to discern genuineness.
